docs/syntax/substitutions.md

```diff
@@ -24,6 +24,8 @@ subs:
 If a substitution is defined globally it may not be redefined (shaded) in a files `frontmatter`. 
 Doing so will result in a build error.
 
+To use the keywords in your files, surround them in curly brackets (`{{key}}`).
```

Comment:
```suggestion
To use the variables in your files, surround them in curly brackets (`{{variable}}`).
```

`keywords` has a more fixed meaning e.g keywords in programmings languages are reserved words (if/else/while etc).